functions/ensure.pp

```diff
@@ -1,4 +1,6 @@
 # @summary function to cast ensure parameter to resource specific value
+#
+# @return [String]
```

Comment:
I think this to be redundant given the `>> String`. Perhaps this deserves a bug report to puppet-strings?

Reply:
Opened https://github.com/puppetlabs/puppet-strings/issues/297 to track this.  For now we have to duplicate the information.